sanic/app.py

```diff
@@ -733,6 +734,47 @@ async def handle_exception(
             context={"request": request, "exception": exception},
         )
 
+        if (
+            request.stream is not None
+            and request.stream.stage is not Stage.HANDLER
+        ):
+            error_logger.exception(exception, exc_info=True)
+            logger.error(
+                "The error response will not be sent to the client for "
+                f'the following exception:"{exception}". A previous response '
+                "has at least partially been sent."
+            )
+
+            # ----------------- deprecated -----------------
+            if self.error_handler._lookup(
+                exception, request.name if request else None
+            ):
+                logger.warning(
+                    "An error occurs during the response process, "
+                    "Sanic no longer execute that exception handler "
+                    "beginning in v22.6 because the error page generated "
+                    "by the exception handler won't be sent to the client "
+                    "because another response was at least partially sent "
+                    "The exception handler should only be used to generate "
+                    "the error page, and if you would like to perform any "
+                    "other action for an exception raised, please consider "
+                    "using a signal handler, like "
+                    '`@app.signal("http.lifecycle.exception")`\n'
+                    "Sanic signals detailed docs: "
+                    "https://sanicframework.org/en/guide/advanced/"
+                    "signals.html"
+                )
+                try:
+                    response = self.error_handler.response(request, exception)
+                    if isawaitable(response):
+                        response = await response
+                except BaseException as e:
+                    logger.error("An error occurs in the exception handler.")
```

Comment:
`occured`

Reply:
Fixed, added `occurred` as the past tense of occurs

Reply:
https://github.com/sanic-org/sanic/pull/2327/commits/a35054715f4a95678d3f73850b623dcb2ed9006c